### ISSUE TITLE
[Buttons] accessibilityFrame doesn't cover the entire view when used with a custom Shape

### ISSUE THREAD

**Issue by codeman7** [material-components/material-components-ios#7527]
MDCButton doesn't update its `accessibilityFrame` when used with a custom shape

## Reproduction steps

1. Turn on VoiceOver
2. Open up the MDCDragons app
3. Navigate to Shaped Buttons example
4. Navigate over the Outlined: Triangular button

## Expected behavior

The `accessibilityFrame` covers the entire button

## Actual behavior

Part of the button is outside the `accessibilityFrame` 

## Additional context

This bug was found within this code snippet
https://github.com/material-components/material-components-ios/blob/22699ace0add8f1921ddb6e9cb6e2d911d4c6c90/components/Buttons/examples/ButtonsShapesExampleViewController.m#L1-L230

![IMG_4143](https://user-images.githubusercontent.com/7131294/58731075-d0fee700-83bb-11e9-8d2a-c807bfe1d95d.PNG)

---
#### Internal data
- Associated internal bug: [b/134167674](http://b/134167674)

**Comment by romoore** [comment]
@codeman7 Did you confirm that the button's tap target is defined by the shape?  I would expect the tappable area and the accessibility frame to be close (if not the same), but the visible shape might be smaller than that.

**Comment by romoore** [comment]
Closing this due to inactivity and lack of confirmation that it's necessary to have the `accessibiltyFrame` include shapes outside of the button's layout frame.

**Comment by Lee-Coderr** [comment]
set accessibilityPath property
